compel/arch/s390/src/lib/infect.c

```diff
@@ -9,6 +9,7 @@
 #include <errno.h>
 #include <elf.h>
 #include <compel/plugins/std/syscall-codes.h>
+#include <linux/const.h>
```

Comment:
Shouldn't this be in compel/arch/s390/src/lib/include/uapi/asm/infect-types.h which contains the `#include <asm/ptrace.h>`?